sdk/core/core-sse/src/utils.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { Readable } from "stream";
```

Comment:
style nit: I'd personally just make a type guard function to check for a "destroy" function and call it rather than pull in node typings just for a cast

Reply:
Good point. I don't like casting here so I changed the input type to be `http.IncomingMessage` to be honest with the client code, although core-rest-pipeline doesn't actually use it as the public type.